### PR TITLE
Fancier caches for merger

### PR DESCRIPTION
**Pull request by bleonard**

### CODE REVIEW

File: tools/merger/data/ci/publish/internal.yml.template [hunk 1]
```diff
@@ -1,10 +0,0 @@
-  publish:
-    docker:
-      - image: circleci/node:14
-        auth:
-          username: $DOCKERHUB_USERNAME
-          password: $DOCKERHUB_PASSWORD
-    steps:
-      - run:
-          name: not publishing
-          command: echo "Not automatically publishing"
```
Comment (evantahler) [inline tools/merger/data/ci/publish/internal.yml.template:10]: Oops! Probably left over from an old template

**Review by evantahler — APPROVED** [review]
This is fancier!  Alphabetical chunks.

Question - why 4 buckets and not 2?  Room to grow?

File: tools/merger/data/ci/publish/internal.yml.template (hunk 1 above)
Comment (bleonard) [reply tools/merger/data/ci/publish/internal.yml.template:10]: I also remove "internal" things.

**Comment by bleonard** [comment]
I'm not sure the size that's correct, so i made them all about 10 each and room to grow ( I think maybe up to 15) so we won't have to think about it for awhile.

**Pull request merged**